packages/react-jsx-highcharts/src/components/PlotBandLine/usePlotBandLineLifecycle.js

```diff
@@ -23,17 +23,13 @@ export default function usePlotBandLine(props, plotType) {
         id: myId,
         ...rest
       };
-
-      if(plotbandline) axis.removePlotBandOrLine(idRef.current);
       setPlotbandline(axis.addPlotBandOrLine(opts, plotType));
```

Comment:
Something strange with axis.addPlotBandOrLine. It does seem to return undefined, but I've seen PlotLineLabel rendering correctly, which depends on it returning object.

Reply:
I suspect this is the bit causing issue, the plot line/band render method [can return undefined](https://github.com/highcharts/highcharts/blob/a188b87b3d2b4f386672ee48e547110e8e61b2a2/js/parts/PlotLineOrBand.js#L127) if `isLine` and `isBand` are false.

Which are defined as
```js
isBand = defined(from) && defined(to)
isLine = defined(value)
```

So potentially `value` is undefined when creating a Plot Line, or either `from` or `to` is undefined when creating a Plot Band?

Reply:
I am actually testing against this: https://stackblitz.com/edit/base-chart-example-olzxfe?file=MyChart.js

The plugin code might actually be hiding the returned object, because the plotline value is actually set.

I am not familiar with highcharts plugins, but I must check this tomorrow without the plugin.

Reply:
It was the plugin code. It is not returning on proceed.apply(....

Reply:
I think I have done all things that I had in mind before next alpha.

Reply:
Cool, happy for me to merge this PR to master?

Reply:
More than happy!